docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+# Logical axiomatization of classes & use of reasoning
+
+This explainer requires understanding of ontology classifications. Please see "an ontology as a classification" section of the [introduction to ontologies documentation](../explanation/intro-to-ontologies.md) if you are unfamiliar with these concepts. 
+
+## What are logical axioms
+
+Logical axioms are relational information about classes that are primarily aimed at machines. This is opposed to annotations like textual definitions which are primarily aimed at humans. These logical axioms allow reasoners to assist in and verify classification, lessening the development burden and enabling expressive queries.
+
+## What should you axiomatize?
+
+Ideally, everything in the definition should be axiomatized when possible. For example, if we consider the cell type `oxytocin receptor sst GABAergic cortical interneuron`, which has the textual definition: 
+
+"An interneuron located in the cerebral cortex that expresses the oxytocin receptor. These interneurons also express somatostatin."
+
+The logical axioms should then follow accordingly:
+
+SubClassOf:
+- interneuron
+- 'has soma location' some 'cerebral cortex'
+- expresses some oxytocin receptor
+- 'capable of' some oxytocin receptor activity
```

Comment:
awks, i think that axiom is in CL -> will make a ticket for it. Will create some figures to show the automatic classification.

Reply:
also from making figures, notice that cortical interneuron has cortex instead of cerebral cortex in equiv class, so it's not classified under cortical interneuron atm >.< have made a ticket to fix that too